updating/index.adoc

```diff
@@ -6,52 +6,52 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can update a {product-title} cluster with a single operation by using the web console or the OpenShift CLI (oc) with {product-title} 4.
+You can update an {product-title} 4 cluster with a single operation by using the web console or the OpenShift CLI (oc).
 
 [id="updating-clusters-overview-understand-update-service"]
 == Understanding the OpenShift Update Service
-xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red{nbsp}Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following information:
+xref:../updating/understanding-the-update-service.adoc#update-service-overview_understanding-the-update-service[Understanding the OpenShift Update Service]: For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. For more information, see the following:
```

Comment:
Why was the non-breaking space in Red Hat removed?

Reply:
May I know the use case of {nbsp}? @nalhadef

Reply:
Hi, @xenolinux. This rule must have been documented somewhere or how else would I know of it? I've requested a discussion about adding this to a style guide. In the meantime, you can skip it if you like. Go ahead and ask for a merge. Thanks!